src/components/task-step/step-with-reading-content.cjsx

```diff
@@ -42,11 +42,14 @@ StepContent = React.createClass
   getCnxId: ->
     TaskStepStore.getCnxId(@props.id)
 
+  shouldExcludeFrame: ->
+    (@props.stepType == 'interactive')
+
```

Comment:
We typically prefer to use: `is` vs `==`, and the parens aren't really needed.  Could be:
```
shouldExcludeFrame: ->
  @props.stepType is 'interactive'
```